source/gloperate/source/viewer/RenderSurface.cpp

```diff
@@ -103,6 +153,18 @@ void RenderSurface::onRender(globjects::Framebuffer * targetFBO)
 {
     cppassist::details() << "onRender()";
 
+    if (m_requestVideo)
```

Comment:
Add a big // [TODO] with an explanation why this is necessary (and what would be better)